gui/src/components/onboarding/pages/body-proportions/ProportionsChoose.tsx

```diff
@@ -1,428 +0,0 @@
-                  <div>
-                    <Typography color="secondary">
-                      {l10n.getString(
-                        'onboarding-choose_proportions-manual_proportions-description'
-                      )}
-                    </Typography>
-                  </div>
-                </div>
-
-                <Button
-                  variant={!state.alonePage ? 'secondary' : 'tertiary'}
-                  to="/onboarding/body-proportions/manual"
-                  className="self-start mt-auto"
-                  state={{ alonePage: state.alonePage }}
-                >
-                  {l10n.getString('onboarding-automatic_proportions-manual')}
-                </Button>
-              </div>
-            </div>
-            {state.alonePage && (
-              <div
-                className={classNames(
-                  'rounded-lg p-4 flex flex-row relative',
-                  !state.alonePage && 'bg-background-70',
-                  state.alonePage && 'bg-background-60'
-                )}
-              >
-                <div className="flex flex-col gap-4">
-                  <div className="flex flex-grow flex-col gap-4 max-w-sm">
-                    <img
-                      onMouseEnter={() => setAnimated(() => true)}
-                      onAnimationEnd={() => setAnimated(() => false)}
-                      src="/images/slimetower.webp"
-                      className={classNames(
-                        'absolute w-[100px] -right-2 -top-24',
-                        animated && 'animate-[bounce_1s_1]'
-                      )}
-                    ></img>
-                    <div>
-                      <Typography variant="main-title" bold>
-                        {l10n.getString(
-                          'onboarding-choose_proportions-auto_proportions'
-                        )}
-                      </Typography>
-                      <Typography variant="vr-accessible" italic>
-                        {l10n.getString(
-                          'onboarding-choose_proportions-auto_proportions-subtitle'
-                        )}
-                      </Typography>
-                    </div>
-                    <div>
-                      <Localized
-                        id="onboarding-choose_proportions-auto_proportions-descriptionv3"
-                        elems={{ b: <b></b> }}
-                      >
-                        <Typography
-                          color="secondary"
-                          whitespace="whitespace-pre-line"
-                        >
-                          Description for autobone
-                        </Typography>
-                      </Localized>
-                    </div>
-                  </div>
-                  <Button
-                    variant="primary"
-                    // Check if we are in dev mode and just let it be used
-                    disabled={beneathFloor && import.meta.env.PROD}
```

Comment:
We still need to consider these for the button to AutoBone on manual proportions now. It should be disabled if there is no headset present or its position is too low. There's also the check if it's a dev environment (for me to test functionality without needing a headset, though I could work around it).